foodsaving/users/migrations/0019_user_photo.py

```diff
@@ -7,13 +7,14 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('users', '0017_auto_20171207_1210'),
+        ('users', '0018_remove_verification_codes'),
```

Comment:
I think you must put new migrations after old migrations that already ran. I fear some problems on staging and master databases when inserting a new migrations between two old ones.